roles/pulp/tasks/install.yml

```diff
@@ -191,6 +180,51 @@
         line: "include-system-site-packages = true"
       when: pulp_use_system_wide_pkgs | bool
 
+    - name: Create requirements.in file to check pulpcore/plugin compatibility
```

Comment:
One general comment: Is this the earliest point in the install that we can feasibly run the check?

I am worried that if the check fails, a system will already have been modified in non-trivial ways.

If it is the earliest point, then it is an acceptable trade-off.

Reply:
We need to run this after we create the virtualenv. That's the only requirement. Do you think we could create the virtualenv even earlier? It's probably possible.

Reply:
For new installs, we could create it a temporary venv earlier perhaps?

For existing installs, we would use the existing venv. And we could use it earlier; before doing any reconfiguration to the system.

I do not think making this change is critical, but if feasible, we should write it up.